core/src/main.js

```diff
@@ -70,6 +70,34 @@ const configReadyCallback = () => {
         return app.$$.ctx.showModal(settings);
       };
 
+      Luigi.openAsSplitView = (path , settings)  => {
```

Comment:
Would it make sense to group it a bit?  like 
```
Luigi.splitView = {
  openAsSplitView: (path , settings) => app.$$.ctx.openSplitView(path, settings),
  closeSplitView: ...
}
```
So its not so cluttered. And you could skip the brackets, so its one line per command.

Reply:
nice suggestion, thanks